sdks/python/apache_beam/testing/util.py

```diff
@@ -261,6 +261,19 @@ def assert_that(
   """
   assert isinstance(actual, pvalue.PCollection), (
       '%s is not a supported type for Beam assert' % type(actual))
+  pipeline = actual.pipeline
+  if getattr(pipeline, 'result', None) is not None:
```

Comment:
Nit: If pipeline is not used elsewhere, just do `getattr(actual.pipeline, 'result', None)`

Reply:
changed

Reply:
un-changed. I realize now we do actually use it in a following block. I have removed the `is None` part though